### PR TITLE
Add events for lambda ALB integration

### PR DESCRIPTION
**Pull request by sjurtf**
The recently added feature of ALBs was to integrate with Lambdas using a target group, and this PR is adding the events needed to this package.

I was working on integrating a Lambda with an ALB and needed to create these events to be able to write my lambda.

I've written them based on documentation page for using Lambda as ALB target.
https://docs.aws.amazon.com/elasticloadbalancing/latest/application/lambda-functions.html#respond-to-load-balancer

Edit:
This fixes parts of #146

### CODE REVIEW

**Comment by qneyrat** [comment]
Can you add test  and md doc ? (you can copy example https://github.com/aws/aws-lambda-go/blob/master/events/apigw_test.go#L13)

**Comment by sjurtf** [comment]
Thank you for taking the time to reply, I'll add tests tomorrow.

File: events/testdata/alb-lambda-target-response.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "headers": {
+    "Set-cookie": "cookies",
+    "Content-Type": "application/json"
+  },
+  "multiValueHeaders": {
+    "Set-cookie": ["cookie-name=cookie-value;Domain=myweb.com;Secure;HttpOnly","cookie-name=cookie-value;Expires=May 8, 2019"],
+    "Content-Type": ["application/json"]
+  },
+  "body": "Hello from Lambda"
+}
```
Comment (qneyrat) [inline events/testdata/alb-lambda-target-response.json]: add new line

File: events/testdata/alb-lambda-target-request.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "User-Agent": ["PostmanRuntime/2.4.5"],
+    "Via": ["1.1 d98420743a69852491bbdea73f7680bd.cloudfront.net (CloudFront)"],
+    "X-Amz-Cf-Id": ["pn-PWIJc6thYnZm5P0NMgOUglL1DYtl0gdeJky8tqsg8iS_sgsKD1A=="],
+    "X-Forwarded-For": ["54.240.196.186, 54.182.214.83"],
+    "X-Forwarded-Port": ["443"],
+    "X-Forwarded-Proto": ["https"]
+  },
+  "isBase64Encoded": false,
+  "body": "Who there?"
+}
```
Comment (qneyrat) [inline events/testdata/alb-lambda-target-request.json]: add new line

**Comment by bmoffatt** [comment]
Thanks for adding the tests. I've made some cosmetic changes to the naming and updated the docs.

**Pull request merged**